docs/data/joy/components/aspect-ratio/FlexRowRatio.js

```diff
@@ -18,10 +18,18 @@ export default function FlexRowRatio() {
         sx={{ gap: 2, minWidth: 300 }}
       >
         <AspectRatio
-          sx={{
-            flexBasis: flexBasis ? `${flexBasis}px` : undefined,
-            overflow: 'auto',
-          }}
+          sx={[
+            {
+              overflow: 'auto',
+            },
+            flexBasis
+              ? {
+                  flexBasis: `${flexBasis}px`,
+                }
+              : {
+                  flexBasis: null,
+                },
+          ]}
```

Comment:
Are all of these sx ternary stuff related to the PR's goal or should we keep it separate? Also, I'm curious why this approach is better than the previous one 🤔

Reply:
Ideally, it can be a separate PR. It was updated due to the `sx` codemod (also transforming `theme.palette.mode` condition).

> I'm curious why this approach is better than the previous one 🤔

It's compatible with Pigment CSS, that's the only benefit. If you don't mind, I can update the sx prop docs in this PR altogether.

Reply:
Ah cool! I'm slightly sad as this syntax is way uglier than the previous one 😅 but it feels all right to update the docs in a dedicated PR. Maybe it's worth highlighting these things (why these ternaries were picked up and missing tasks) in the PR's description, though!